### ISSUE TITLE
Funding for chart.xkcd

### ISSUE THREAD

**Issue by chrisvxd** [timqian/chart.xkcd#55]
Hey @timqian! 👋

Just wanted to reach out about funding your work on `chart.xkcd`.

@transitive-bullshit and I are both open source maintainers, and we're exploring new ways to make OSS funding a viable source of income.

We're currently looking at building rate-limited SaaS APIs on top of existing OSS libraries, reserving the majority of profits for the original authors. [We've built a few](https://saasify.sh/#/showcase) at this point, but want to keep adding more.

I think `chart.xkcd` would be a great fit, and I'd love to use it to build a SaaS API for rendering graphs to images. We'd handle dev, auth, billing and support so you wouldn't have to do anything.

Looking forward to hearing your thoughts! 🙏

**Comment by transitive-bullshit** [comment]
I'm incredibly biased since I'm working with Chris, but I have to say that if we work out the specifics, the ability to include xkcd-style charts as images inline in HTML as opposed to bundling a JS script would be incredibly powerful.

Think of how much easier this would make certain uses cases where people want to use cool libs like this but either don't have the knowledge or access to the underlying implementation in order to bundle the required libraries.

Just a thought.

**Comment by timqian** [comment]
Yeah, I think an API to render data to an image should be useful. Looking forward to your work!

**Comment by chrisvxd** [comment]
Awesome, thanks for the green light @timqian! I'll keep you posted via this thread.

If you're interested, we also have [an open slack](https://join.slack.com/t/saasify/shared_invite/enQtODAxODA5MzU0NjczLTczOGU3NzNkYTJlMWIwZDkyNjJkOTk3MGEwZThlOWQyNTQxODZjZTExNjAzODJlZDQ3MWM5NWQwMGRiMDcyZTY) where you can reach us more directly.
